torchvision/utils.py

```diff
@@ -88,8 +91,9 @@ def norm_range(t, range):
     return grid
 
 
-def save_image(tensor, fp, nrow=8, padding=2,
-               normalize=False, range=None, scale_each=False, pad_value=0, format=None):
+def save_image(tensor: Union[torch.Tensor, Sequence[torch.Tensor]], fp: Union[str, io.FileIO, io.BytesIO],
```

Comment:
I wonder if there is a canonical way of writing this, as PIL also support `Path` objects https://github.com/python-pillow/Pillow/blob/349bcd86915958f67b865f5c4c9ef683b7b46756/src/PIL/Image.py#L2050

Reply:
Maybe I misunderstand you, but we could simply add `pathlib.Path` to the list. If we need to do this more often we can simply create an alias `PILFp = Union[str, io.FileIO, io.BytesIO, pathlib.Path]` and use this within the function signature instead.

Reply:
Yes, we can simply add `pathlib.Path` to the list, but my thinking would be to do whatever PIL does here (as we simply dispatch to PIL anyway).

For example, in a PIL WIP PR they do it as follows: https://github.com/python-pillow/Pillow/pull/2941/files#diff-b49fc26d17e5d3b539a05c7850abe2b0R2025

```
Union[Text, pathlib.Path, BinaryIO]
```

But I don't have any strong opinions here.

Reply:
Sure, I agree we should use the same annotations as `PIL`. I wasn't aware that they begun adding them. Thanks for the heads up!